v_4_9_1/files/config/kube-proxy.yaml

```diff
@@ -5,4 +5,7 @@ kind: KubeProxyConfiguration
 mode: iptables
 resourceContainer: /kube-proxy
 clusterCIDR: {{.Cluster.Calico.Subnet}}/{{.Cluster.Calico.CIDR}}
+{{- if .Cluster.KubeProxy.ConntrackMaxPerCore }}
+conntrackMaxPerCore: {{.Cluster.KubeProxy.ConntrackMaxPerCore}}
```

Comment:
Does this actually work? Where is `ConntrackMaxPerCore` actually set?

Reply:
We need to set up `ConntrackMaxPerCore` outside of the k8scc